src/engine/instrument.h

```diff
@@ -377,6 +389,13 @@ struct DivInstrumentAmiga {
   }
 };
 
+struct DivInstrumentX1_010 {
+  int bankSlot;
+
+  DivInstrumentX1_010():
+    bankSlot(0) {}
```

Comment:
Curious - how would we implement this?

Reply:
there's 2 possible way:
Separated chip ID for Bankswitched one: because it supports expanded ROM size
or 
Flags to toggle bankswitch mode - needs to reallocate or extend sample memory area